app/forms/validation/mappings/Formatters.scala

```diff
@@ -39,9 +39,9 @@ trait Formatters {
 
   private[mappings] def currencyFormatter(requiredKey: String,
                                           invalidNumericKey: String,
-                                          nonNumericKey: String,
                                           maxAmountKey: String,
-                                          args: Seq[String] = Seq.empty): Formatter[BigDecimal] =
+                                          args: Seq[String] = Seq.empty[String]
```

Comment:
is nonNumericKey not used anywhere then?

Reply:
Not here.  I think it used to be used for the message Enter numbers 0-9 etc. Like below in personal

Reply:
`
  def amountForm(
                  emptyFieldKey: String,
                  wrongFormatKey: String = "common.error.invalid_currency_format",
                  exceedsMaxAmountKey: String = "common.error.amountMaxLimit",
                  emptyFieldArguments: Seq[String] = Seq.empty[String]
                ): Form[BigDecimal] = Form(
    amount -> currency(
      requiredKey = emptyFieldKey,
      invalidNumeric = wrongFormatKey,
      nonNumericKey = wrongFormatKey,
      maxAmountKey = exceedsMaxAmountKey,
      args = emptyFieldArguments
    )
  )`